httomo/monitors/__init__.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, List, Optional
+from httomo.monitors.aggregate import AggregateMonitoring
+from httomo.monitors.benchmark import BenchmarkMonitoring
+from httomo.monitors.summary import SummaryMonitor
+from httomo.runner.monitoring_interface import MonitoringInterface
+
+
+MONITORS_MAP = {
+    "bench": BenchmarkMonitoring,
+    "summary": SummaryMonitor
+}
+
+def make_monitors(monitor_descriptors: List[str]) -> Optional[MonitoringInterface]:
+    if len(monitor_descriptors) == 0:
+        return None
+    
+    monitors: List[MonitoringInterface] = []
+    for descriptor in monitor_descriptors:
+        if descriptor not in MONITORS_MAP:
+            raise ValueError(f"Unknown monitor '{descriptor}'. Please choose one of {MONITORS_MAP.keys}")
```

Comment:
Nitpick, but I think `MONITORS_MAP.keys` is a method, so it needs to be executed by adding the parentheses:
```suggestion
            raise ValueError(f"Unknown monitor '{descriptor}'. Please choose one of {MONITORS_MAP.keys()}")
```